drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -167,15 +165,15 @@ def items_without_answers():
             "display_zone_labels": self.data.get('displayLabels', False),
             "display_zone_borders": self.data.get('displayBorders', False),
             "items": items_without_answers(),
-            "title": self.display_name,
+            "title": self.ugettext(self.display_name),
             "show_title": self.show_title,
             "problem_text": self.question_text,
             "show_problem_header": self.show_question_header,
             "target_img_expanded_url": self.target_img_expanded_url,
-            "target_img_description": self.target_img_description,
-            "item_background_color": self.item_background_color or None,
-            "item_text_color": self.item_text_color or None,
-            "initial_feedback": self.data['feedback']['start'],
+            "target_img_description": self.ugettext(self.target_img_description),
+            "item_background_color": self.ugettext(self.item_background_color) or None,
+            "item_text_color": self.ugettext(self.item_text_color) or None,
```

Comment:
I would add: as a general rule, any time you are wrapping `ugettext` around something other than a constant string is probably not correct. AFAIK, the only time it is correct is when it's translating a value that could not possibly be translated earlier and where all possible values of that variable are guaranteed to have been previously marked with `ugettext_noop()`. A lot of the uses below look wrong to me, since they are wrapping user-provided strings.

Example from my [previous comment](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/59#discussion_r53235814): 'Translating user-provided content doesn't just have the problem of missing translations; it can also cause problems. e.g. French instructor writes "Location" (which means "Rental" in French), but after getting translated by this code, it would appear in the course as "Emplacement" or something, which has a completely different meaning.'